### ISSUE TITLE
cert: creating new account does not "overwrite" and create a new identity, it rather creates a new visible string, but the "new user" can still access the channel of the previous user as they still share the same cert.

### ISSUE THREAD

**Issue by haschimoto** [zer0net/ifs-master#228]

**Comment by dnelband** [comment]
ok the issue is this, every client has a unique user id generated for every zite, this unique user ID is generated once, and the creating a new IFS Certificate on the client doesnt trigger a regenration. there is nothing in the docs about this so i wrote tamas about this.

**Comment by haschimoto** [comment]
We had this working and solved before. I know because i tested this.
It created various new identities that you could easily see by chosing it
from the sidebar also.

**Comment by dnelband** [comment]
yeah i know, but something has changed because i didnt change anything in the way certificates are created, so either this logic was changed or i need to update something that i dont know of.
